internal/dag/status_test.go

```diff
@@ -804,6 +804,24 @@ func TestDAGStatus(t *testing.T) {
 		},
 	}
 
+	proxyValidReuseCaseExampleCom := &contour_api_v1.HTTPProxy{
```

Comment:
Should this test be part of a separate PR?

Reply:
This is part of https://github.com/projectcontour/contour/pull/3231 which has been approved but not been merged yet.  Looks like because that was on the master branch it gets included here so I'll try to remember not to do that next time.

Reply:
I think you'll have to do a rebase and remove the commits referencing the case-insensitive fqdn code (same as above comment)

Reply:
Thanks @sunjayBhatia, that did the trick.